src/html2ans/parsers/text.py

```diff
@@ -245,7 +246,7 @@ class LinkParser(BaseElementParser):
     applicable_elements = ['a']
 
     def parse(self, element, *args, **kwargs):
-        result = self.construct_output(element, "interstitial_link", element.text)
+        result = self.construct_output(element, "interstitial_link", str(element))
```

Comment:
This will include html in the `content` of the `interstitial_link`, which is incorrect.